tools/chapel-py/src/method-tables/uast-methods.h

```diff
@@ -29,18 +29,19 @@
 //
 
 CLASS_BEGIN(AstNode)
+  ITER_PROTOTYPE(AstNode, const chpl::uast::AstNode*)
```

Comment:
This iter prototype makes me think we should support something similar for __str__, which suffers the same problems as __iter__

Reply:
I agree. In the meantime, while I think about a more general way to make this work, I think the iter prototype is an improvement over what we have.

Reply:
I definitely agree, not suggesting you add something for `__str__` in this PR